WordPressAuthenticator/GoogleSignIn/GoogleOAuthTokenGetter.swift

```diff
@@ -8,11 +8,17 @@ class GoogleOAuthTokenGetter: GoogleOAuthTokenGetting {
 
     func getToken(
         clientId: GoogleClientId,
+        audience: String,
```

Comment:
This is not in the spec at https://developers.google.com/identity/protocols/oauth2/native-app#step-2:-send-a-request-to-googles-oauth-2.0-server but we'll get an idToken that our backend considers invalid if omitted.

It is, however, in another piece of documentation that I didn't think of writing down. Interestingly, wherever I found out about this used the name "audience" but in our existing code the value assigned to the SDK equivalent is `googleLoginServerClientId`.